### PR TITLE
allows managers to access embargoed objects

### PR DESCRIPTION
**Pull request by ajstanley**

### CODE REVIEW

File: islandora_ip_embargo.module [hunk 1]
```diff
@@ -138,7 +138,7 @@ function islandora_ip_embargo_islandora_datastream_access($op, $islandora_datast
 function islandora_ip_embargo_islandora_object_access($op, $islandora_object, $user) {
   module_load_include('inc', 'islandora_ip_embargo', 'includes/utilities');
   // Handle an embargo.
-  if (islandora_ip_embargo_restrict_access($islandora_object->id)) {
+  if (islandora_ip_embargo_restrict_access($islandora_object->id) && !user_access(ISLANDORA_IP_EMBARGO_MANAGE_EMBARGOES)) {
```
Comment (willtp87) [inline islandora_ip_embargo.module:141]: Will this not allow managers access when they are outside the white listed IPs?

Comment (ajstanley) [inline islandora_ip_embargo.module:141]: David and I talked about this.  The idea is that managers would be able to manage their objects when they were away from their whitelisted ip.  Users from the restricted areas would not have managerial permissions.

Comment (MorganDawe) [inline islandora_ip_embargo.module:141]: I tested this as user 1 and an authenticated user with ip embargo management permissions while being outside the white zone list and i was not allowed to view the object.

Comment (dwilcox) [inline islandora_ip_embargo.module:141]: I have an email out to the client to ask about desired behaviour. I'll post back when I hear from them.

**Comment by willtp87** [comment]
This should be reverted until we hear back from the sponsor of this fix.

**Comment by dwilcox** [comment]
I just heard back from the client and I can confirm that a user with permission to manage IP Embargoes should still be able to manage an embargoed object if they are outside the allowed list of IP addresses. This wasn't clear in the initial module specification, but since they have confirmed the desired behaviour we can keep this pull request merged in and proceed with QA.

**Pull request merged**